mock-data/CosmosDbSeeder/items/index_data/notification_config.json

```diff
@@ -0,0 +1,38 @@
+{
+  "id": "notification_config",
+  "docType": "notification_configuration",
```

Comment:
docType should be "system" - this allows us to have a single partition for all the system level documents